app/scripts/metamask-controller.js

```diff
@@ -723,6 +762,7 @@ export default class MetamaskController extends EventEmitter {
       setUseBlockie: this.setUseBlockie.bind(this),
       setUseNonceField: this.setUseNonceField.bind(this),
       setUsePhishDetect: this.setUsePhishDetect.bind(this),
+      setUseStaticTokenList: this.setUseStaticTokenList.bind(this),
```

Comment:
I *think* you can get the same effect by doing

```suggestion
      setUseStaticTokenList: nodeify(
        this.preferencesController.setUseStaticTokenList,
        this.preferencesController
      ),
```

Reply:
If this suggestion is taken and works then the other method added to the metamask-controller can be removed.